README.md

```diff
@@ -61,6 +61,15 @@ class MySchema < GraphQL::Schema
 end
 ```
 
+**Optional:** To opt in to Federation v2, specify the version in your schema:
+
+```ruby
+class MySchema < GraphQL::Schema
+  include ApolloFederation::Schema
+  federation version: 2.0
```

Comment:
Might this be the place to support `import`?

Reply:
`version:` should probably be a string. we don't foresee a need for `2.0.1` but it's possible